TrappingRainWater/main.h

```diff
@@ -0,0 +1,11 @@
+#ifndef MAIN_H
+#define MAIN_H
+
+#include <vector>
+
+class Solution {
```

Comment:
what's the purpose of a class with no data members and just one function? 
(This may be a requirement of some grading system, but I am ignorant about those)